COPYING

```diff
@@ -5,6 +5,7 @@ Copyright (C) 2002-present Jason Evans <jasone@canonware.com>.
 All rights reserved.
 Copyright (C) 2007-2012 Mozilla Foundation.  All rights reserved.
 Copyright (C) 2009-present Facebook, Inc.  All rights reserved.
+Copyright (C) 2024 Arm Ltd. All rights reserved.
```

Comment:
Thanks for the PR however we are not inclined to change the copyright right now. Feel free to reopen the PR if you can submit w/o this change.